client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -152,24 +201,226 @@ def process(self, instance):
+            ),
+            "uvWrite": (
+                "Uv data for PolyMesh and SubD shapes will be written to the "
+                "Alembic file."
+            ),
+            "uvsOnly": (
+                "If this flag is present, only uv data for PolyMesh and SubD "
+                "shapes will be written to the Alembic file."
+            ),
+            "verbose": "Prints the current frame that is being evaluated.",
+            "wholeFrameGeo": (
+                "Data for geometry will only be written out on whole frames."
+            ),
+            "worldSpace": "Any root nodes will be stored in world space.",
+            "writeColorSets": "Write vertex colors with the geometry.",
+            "writeFaceSets": "Write face sets with the geometry.",
+            "writeNormals": "Write normals with the deforming geometry.",
+            "writeUVSets": (
+                "Write all uv sets on MFnMeshes as vector 2 indexed geometry"
+                " parameters with face varying scope."
+            ),
+            "writeVisibility": (
+                "Visibility state will be stored in the Alembic file. "
+                "Otherwise everything written out is treated as visible."
+            )
+        }
+        tooltip = ""
+        for flag in flags:
+            tooltip += "{} - {}\n".format(flag, tooltips[flag])
+        defs.append(
+            EnumDef(
+                "flags",
+                flags,
+                default=enabled_flags,
+                multiselection=True,
+                label="Export Flags",
+                tooltip=tooltip,
+            )
+        )
+
+        for key, value in override_defs.items():
+            if key not in overrides:
+                continue
+
+            defs.append(value)
+
+        defs.append(
+            UISeparatorDef("sep_alembic_options")
+        )
+
+        return defs
+
 
 class ExtractAnimation(ExtractAlembic):
-    label = "Extract Animation"
+    label = "Extract Animation (Alembic)"
     families = ["animation"]
 
     def get_members_and_roots(self, instance):
-
         # Collect the out set nodes
         out_sets = [node for node in instance if node.endswith("out_SET")]
         if len(out_sets) != 1:
-            raise RuntimeError("Couldn't find exactly one out_SET: "
-                               "{0}".format(out_sets))
+            raise RuntimeError(
+                "Couldn't find exactly one out_SET: {0}".format(out_sets)
+            )
         out_set = out_sets[0]
-        roots = cmds.sets(out_set, query=True)
+        roots = cmds.sets(out_set, query=True) or []
 
         # Include all descendants
-        nodes = roots + cmds.listRelatives(roots,
-                                           allDescendents=True,
-                                           fullPath=True) or []
+        nodes = (
+            roots
+            + cmds.listRelatives(roots, allDescendents=True, fullPath=True)
+            or []
+        )
```

Comment:
To me this new code is way more unreadable. Anyone else has that?

Reply:
Yup, agree. Suggested something else now.